app/views/staff/proposals/_form.html.haml

```diff
@@ -10,11 +10,12 @@
       %section.inline-block
         = f.label :slides_url
         = f.text_field :slides_url, class: "form-control"
-      %p
-      = f.select :review_tags,
-        options_for_select(event.review_tags, proposal.object.review_tags),
-        {}, {class: 'multiselect review-tags', multiple: true}
-
+  .row
```

Comment:
I don't think we want to add a new row and fieldset here.  Also, the %section below might no display the same as the original %p, so verify visually it still looks correct.